runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/actions/excel.ts

```diff
@@ -3,12 +3,16 @@ import englishExcelActionsTranslations from '#src-app/translations/en/process/ac
 const excelActionsTranslations: typeof englishExcelActionsTranslations = {
     'Process.Details.Modeler.Actions.Excel.Row': 'Wiersz',
     'Process.Details.Modeler.Actions.Excel.Column': 'Kolumna',
+    'Process.Details.Modeler.Actions.Excel.StartRow': 'Wiersz początkowy',
+    'Process.Details.Modeler.Actions.Excel.StartColumn': 'Kolumna początkowa',
     'Process.Details.Modeler.Actions.Excel.Worksheet': 'Arkusz',
     'Process.Details.Modeler.Actions.Excel.Open.Label': 'Otwórz',
     'Process.Details.Modeler.Actions.Excel.Open.Path.Title': 'Ścieżka / Skoroszyt',
     'Process.Details.Modeler.Actions.Excel.Open.Worksheet.Title': 'Arkusz',
     'Process.Details.Modeler.Actions.Excel.GetCell.Label': 'Odczytaj komórkę',
     'Process.Details.Modeler.Actions.Excel.SetCell.Label': 'Wpisz do komórki',
+    'Process.Details.Modeler.Actions.Excel.SetCells.Label': 'Wpisz do komórek',
```

Comment:
'Dodaj do komórki' sound a little bit better imo

Reply:
then in en translations should be "Add cell value" instead of current "set cell value"
for me "dodaj do" is misleading because it suggests adding something to cell value or even e.g. adding column next to it. 

imo "Ustaw wartość komórek"?